### ISSUE TITLE
Starcounter 2.3 issue

### ISSUE THREAD

**Issue by iliassays** [Starcounter/Starcounter#4175]
Hello

See the attachment. I am having this on my local machine once i install 2.3. What is the soln?

![scissue](https://cloud.githubusercontent.com/assets/3203987/24192703/08eaa4c4-0f1a-11e7-9cf6-232b781942f4.PNG)

**Comment by miyconst** [comment]
Please provide full Starcounter version.

cc @alemoi

**Comment by iliassays** [comment]
Starcounter-RC-2.3.0.5971-Setup. I installed this.

**Comment by alemoi** [comment]
Hi Ilias! Does it t happen with default Launcher app? No changes?

**Comment by alemoi** [comment]
Was it working before? Is it a new computer?

**Comment by iliassays** [comment]
No. It is happening while running anything for starcounter. Even while trying to browse http://127.0.0.1:8181/

**Comment by alemoi** [comment]
Wow, so something is really wrong with clang.
Is it a new environment?

**Comment by iliassays** [comment]
No. I was using SC 2.2. Now installed 2.3.

**Comment by alemoi** [comment]
Thanks! I really want to find whats the issue! Can you please zip folder `%temp%\starcounter` and send it here? Thanks!

**Comment by iliassays** [comment]
[2.2.5.zip](https://github.com/Starcounter/Starcounter/files/861465/2.2.5.zip)

**Comment by alemoi** [comment]
Thanks! I checked it. Can you please stop Starcounter, delete this folder, start Starcounter, get the error, zip the folder again and send it. Thanks so much!

**Comment by iliassays** [comment]
Here we go!
[2.2.5.zip](https://github.com/Starcounter/Starcounter/files/861563/2.2.5.zip)

**Comment by alemoi** [comment]
So after you have deleted this folder, it was still crashing I suppose?

**Comment by alemoi** [comment]
Is it crashing now, even if you simply start `scservice.exe` in cmd, without any applications?

**Comment by iliassays** [comment]
was that the solution :)

It is not crashing now and i can see http://127.0.0.1:8181/

**Comment by alemoi** [comment]
What if you start an app, would it crash in that case?

**Comment by iliassays** [comment]
no. working fine now. What was the issue?

**Comment by alemoi** [comment]
I will have to investigate, thanks to you I have old files that were cached.
